locales/lt-LT/taoResultServer.rdf.po

```diff
@@ -63,12 +63,12 @@ msgstr "Duomenys apie saugyklą yra išsiųsti į žurnalą"
 # http://www.tao.lu/Ontologies/TAOResultServer.rdf#implementation
 msgctxt "http://www.w3.org/2000/01/rdf-schema#label"
 msgid "Link to the implementation Class"
-msgstr ""
+msgstr "Link to the implementation Class"
```

Comment:
Why do you add English translation here?

Reply:
it was in the translation source files, I did not pay attention. It fixed